apps/src/blockly/themes/accessibleColors.mjs

```diff
@@ -58,12 +63,22 @@ var accessiblePalette = {
   color_FF4235: '#FF4235'
 };
 
+console.log(
+  '\x1b[33m%s\x1b[0m',
+  'Copy the following values into cdoAccessible.js:'
+);
+
 // Each color in our standard palette is mapped to a new "nearest" color from the accesible palette.
 for (const [, value] of Object.entries(blockStyles)) {
-  const nearestColor = require('nearest-color').from(accessiblePalette);
-  value.colourPrimary = nearestColor(value.colourPrimary).value;
+  const nearestAvailableColor = nearestColor.from(accessiblePalette);
+  value.colourPrimary = nearestAvailableColor(value.colourPrimary).value;
   // Remove the color from the available palette so we don't assign it twice.
   delete accessiblePalette['color_' + value.colourPrimary.slice(1)];
 }
 
-export default blockStyles;
+console.log(blockStyles);
+console.log(
+  '\x1b[33m%s\x1b[0m',
```

Comment:
This just tells terminal to use yellow text to better stand out against what is being copied.
https://stackoverflow.com/questions/9781218/how-to-change-node-jss-console-font-color